yt/frontends/athena_pp/data_structures.py

```diff
@@ -266,11 +266,13 @@ def __init__(
             unit_system=unit_system,
             default_species_fields=default_species_fields,
         )
-        self.filename = filename
         if storage_filename is None:
-            storage_filename = f"{filename.split('/')[-1]}.yt"
+            storage_filename = self.basename + ".yt"
         self.storage_filename = storage_filename
-        self.backup_filename = self.filename[:-4] + "_backup.gdf"
+
+    @property
+    def backup_filename(self):
+        return self.filename[:-4] + "_backup.gdf"
```

Comment:
this looks like a bad copy-pasta from the Athena frontend, I should avoid propagating the bug and align with my refactor in Athena (here we don't want to strip 4 chars but 5 or 6, as the extension is not ".vtk" but ends with athdf, see _is_valid method def here)